test/modules/map/components/olMap/handler/measure/OlMeasurementHandler.test.js

```diff
@@ -99,9 +102,56 @@ describe('OlMeasurementHandler', () => {
 			expect(map.removeOverlay).toHaveBeenCalledTimes(4);
 			expect(classUnderTest._overlays.length).toBe(0);
 		});	
+
+		
 	});
 
+	describe('when activated over olMap', () => {
+		
+		const initialCenter = fromLonLat([11.57245, 48.14021]);
+
+		const setupMap =  () => {
+			return new Map({
+				layers: [
+					new TileLayer({
+						source: new OSM(),
+					}),
+					new TileLayer({
+						source: new TileDebug(),
+					})],
+				target: 'map',
+				view: new View({
+					center: initialCenter,
+					zoom: 1,
+				}),
+			});
+			
+		};
+
+		it('uses EnvironmentService for snapTolerance', () => {
+			const classUnderTest = new OlMeasurementHandler();			
+			const environmentSpy = spyOn(environmentServiceMock, 'isTouch');
+			const map =  setupMap();
+
+			classUnderTest.activate(map);
+			expect(environmentSpy).toHaveBeenCalled();			
+		});
 
+		it('isTouch() resolves in higher snapTolerance', () => {
+			const classUnderTest = new OlMeasurementHandler();
+			environmentServiceMock.isTouch = () => true;
```

Comment:
It's often better to use a spy  to define returned values:
`spyOn(environmentServiceMock, 'isTouch').and.returnValue(true);`

Then we can also test for call parameters and call counts if needed...